packages/framer-motion/src/render/VisualElement.ts

```diff
@@ -397,7 +397,11 @@ export abstract class VisualElement<
             this.removeFromVariantTree = this.parent.addVariantChild(this)
         }
 
-        this.values.forEach((value, key) => this.bindToMotionValue(key, value))
+        this.values.forEach((value, key) => {
+            if (!this.valueSubscriptions.has(key)) {
+                this.bindToMotionValue(key, value)
+            }
```

Comment:
I also don't fully understand this change, as `bindToMotionValue` seems to be designed to handle cases where a value subscription is present as well. 🤔

Reply:
Ah good catch - this is a legacy from fixing the same bug that other check fixed (that I discovered while making this PR)